cf-custom-resources/lib/custom-domain.js

```diff
@@ -158,17 +163,28 @@ const writeARecord = async function (
     hostedZoneCache.set(domain, hostedZoneId);
   }
   console.log(`${action} A record into Hosted Zone ${hostedZoneId}`);
-  const changeBatch = await updateRecords(
-    route53,
-    hostedZoneId,
-    action,
-    alias,
-    accessDNS,
-    accessHostedZone
-  );
-  await waitForRecordChange(route53, changeBatch.ChangeInfo.Id);
+  try {
+    const changeBatch = await updateRecords(
+      route53,
+      hostedZoneId,
+      action,
+      alias,
+      accessDNS,
+      accessHostedZone
+    );
+    await waitForRecordChange(route53, changeBatch.ChangeInfo.Id);
+  } catch (err) {
+    if (action === changeRecordAction.Delete && isRecordSetNotFoundErr(err)) {
+      console.log(`${err.message}; Copilot is ignoring this record.`);
```

Comment:
How about keeping `Copilot is ignoring this record.` at the beginning so that customers see our message first before seeing long less user-friendly error?

Reply:
Ahh I see the exact error message below which is also not bad. It is pretty clear. You can skip the above comment